cli/module_build.go

```diff
@@ -372,3 +411,63 @@ func jobStatusFromProto(s buildpb.JobStatus) jobStatus {
 		return jobStatusUnspecified
 	}
 }
+
+// envOrValue does os.Getenv if input starts with $, otherwise returns value directly.
+func envOrValue(orig string) string {
```

Comment:
Spoke offline; I'd rather avoid this env-pulling functionality for now if we're just using existing auth functionality.